tests/arch/arm/arm_thread_swap/src/arm_thread_arch.c

```diff
@@ -334,7 +340,7 @@ static void alt_thread_entry(void)
 	 */
 #if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
 	__asm__ volatile (
-		"push {r4,r5,r6,r7};\n\t"
+		"push {r4, r5, r6, r7};\n\t"
```

Comment:
Nit: can be simplified
```suggestion
		"push {r4-r7};\n\t"
```

Reply:
ack